---
layout: paper
date: April 29, 2022
title: Reference-free cell type deconvolution of multi-cellular pixel-resolution spatially resolved transcriptomics data
authors: <b>Brendan F. Miller</b>, Feiyang Huang, Lyla Atta, Arpan Sahoo, Jean Fan
image:
abstract: <br>Recent technological advancements have enabled spatially resolved transcriptomic profiling but at multi-cellular pixel resolution, thereby hindering the identification of cell-type-specific spatial patterns and gene expression variation. To address this challenge, we develop STdeconvolve as a reference-free approach to deconvolve underlying cell types comprising such multi-cellular pixel resolution spatial transcriptomics (ST) datasets. Using simulated as well as real ST datasets from diverse spatial transcriptomics technologies comprising a variety of spatial resolutions such as Spatial Transcriptomics, 10X Visium, DBiT-seq, and Slide-seq, we show that STdeconvolve can effectively recover cell-type transcriptional profiles and their proportional representation within pixels without reliance on external single-cell transcriptomics references. STdeconvolve provides comparable performance to existing reference-based methods when suitable single-cell references are available, as well as potentially superior performance when suitable single-cell references are not available. STdeconvolve is available as an open-source R software package with the source code available at https://github.com/JEFworks-Lab/STdeconvolve.
journal: Nature Communications
journalurl: https://www.nature.com/articles/s41467-022-30033-z
pubmedurl:
pdfurl: pdfs/2022-04-29-STdeconvolve.pdf
code: https://github.com/JEFworks-Lab/STdeconvolve
categories: [Computational]
---
